src/MatBlazor/Components/MatSlider/BaseMatSlider.cs

```diff
@@ -82,13 +82,10 @@ public string MarkerStyle
             {
                 try
                 {
-                    decimal min = 0;
-                    decimal.TryParse(ValueMin.ToString(), out min);
-                    decimal max = 0;
-                    decimal.TryParse(ValueMax.ToString(), out max);
-                    decimal step = 1;
-                    decimal.TryParse(Step.ToString(), out step);
-                    return "background: linear-gradient(to right, currentcolor 2px, transparent 0px) 0px center / calc((100% - 2px) / " + ((max - min) / step).ToString() + ") 100% repeat-x;";
+                    decimal.TryParse(ValueMin.ToString(), out var min);
+                    decimal.TryParse(ValueMax.ToString(), out var max);
+                    decimal.TryParse(Step.ToString(), out var step);
+                    return "background: linear-gradient(to right, currentcolor 2px, transparent 0px) 0px center / calc((100% - 2px) / " + ((max - 0) / step).ToString() + ") 100% repeat-x;";
```

Comment:
@stefanloerwald  I've not really worked with decimal datatype, but I suspect that this change _might_ lead to a DivisionByZero error unless it's guarded somewhere else?

Reply:
Good catch! The original code was invalid!
```cs
decimal step = 1;
decimal.TryParse(Step.ToString(), out step);
```
This will either result in `step = default` or, if `Step.ToString()` parses as a decimal, in some value that is represented by `Step`. However, the value "1" is in any case overwritten.

This must actually be:
```cs
if (!decimal.TryParse(Step.ToString(), out var step))
{
    step = 1;
}
```